### DISCUSSION TITLE
Discord invite invalid?

### DISCUSSION THREAD

**Discussion by zeebear** [codewars/codewars.com#2511] [Q&A] [answered]
I'm late to the party, I just discovered that Codewars is on Discord -- but the invitation links in the emails and in the old forum on Codewars are invalid. Any chance to get a new Discord invite link?

**Comment by hobovsky** [comment, marked as answer]
Which invites did you try? Did you try this one: https://discord.gg/mSwJWRvkHA ?

**Reply by zeebear** [reply]
That is the same one that I tried before and it didn't work -- now it does work? Not sure what happened, but I'm in now -- thanks!
